docs/testbenches/project_based/template/index.rst

```diff
@@ -8,33 +8,44 @@ Project based test bench template
 Overview
 -------------------------------------------------------------------------------
 
+**\*This section must contain: the purpose of the testbench/ type of interface
+it's validating, the corresponding hdl project and the HDL Github documentation
+page.**\ \*
+
 Block design
 -------------------------------------------------------------------------------
 
+**\*Mention the HDL and SV components of the testbench .**\ \*
+
 Block diagram
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-..
-   MUST: Use SVG format for the diagram
+If the testbench has multiple ways of configuration, then make subsections to
+this section and show the default configuration and some other popular modes.
 
-..
-   TIP: Block diagrams should contain subtitles only if there are at least two
-   different diagrams
+\**\* KEEP THIS PARAGRAPH \**\*
+The data path and clock domains are depicted in the below diagram:
 
 .. image:: ../template/project_based_template_bd.svg
    :width: 800
    :align: center
    :alt: Template/Testbench block diagram
 
+\*\* MUST: Use SVG format for the diagram \*\*
+
+\*\* TIP: Block diagrams should contain subtitles only if there are at least two
+different diagrams \*\*
+
 Configuration parameters and modes
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-..
-   MENTION IF ANY MODES ARE AVAILABLE FOR CONFIGURATION
+\**\* MENTION IF ANY MODES ARE AVAILABLE FOR CONFIGURATION \**\*
```

Comment:
MENTION IF THERE ARE ANY CONFIGURATION PARAMETERS AND/OR MODES

Reply:
Updated.